try.html

```diff
@@ -529,6 +529,14 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
   <td><img src='/gem/u/raphink.svg' alt=''/></td>
   <td><code>https://img.shields.io/gem/u/raphink.svg</code></td>
   </tr>
+  <tr><th> APIstatus.org: </th>
+  <td><img src='/apistatus/o/aHR0cHM6Ly9tb2NrYmluLmNvbS9zdGF0dXMvMjAw.svg' alt=''/></td>
```

Comment:
Originally I tried to put the URL into the shields.io URL but that caused problems with the routing so I had to come up with a workaround. That's also why I built the apistatus.org website to generate the base64 encoded URL. Looking at wercker and codeship's integrations they also have unique hashes instead of a more straightforward approach, and I think the premise is that you will need to be familiar with the service behind the badge before adding it to your project. 

The difference between the badges should be pretty apparent by seeing the output of the badge (online vs 301 redirect) alongside the URLs but I could add a better description in the label.